### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by mtazzari**
Using issue templates to help responding to the issues more quickly.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG] "
+labels: bug
```
Comment (jeffjennings) [inline .github/ISSUE_TEMPLATE/bug_report.md:5]: Is it worth giving a list of possible labels? e.g. , 'bug', 'requested_expansion', 'question'

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+assignees: ''
```
Comment (jeffjennings) [inline .github/ISSUE_TEMPLATE/bug_report.md:6]: Maybe also give a list? "Leave blank unless you want to suggest specifically @rbooth200, @jeffjennings and/or @mtazzari."

**Review by jeffjennings — COMMENTED** [review]
cool! I see [the default](https://docs.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository) and think your version is even better!

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (mtazzari) [reply .github/ISSUE_TEMPLATE/bug_report.md:5]: Good idea:
```suggestion
labels: bug [e.g.: 'bug', 'enhancement', 'question']
```

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (mtazzari) [reply .github/ISSUE_TEMPLATE/bug_report.md:6]: Good idea!
```suggestion
assignees: '' [Leave blank unless you want to suggest specifically @rbooth200, @jeffjennings and/or @mtazzari.]
```

**Review by jeffjennings — APPROVED** [review]

**Comment by rbooth200** [comment]
Is there a reason this PR has not been merged?

**Comment by jeffjennings** [comment]
don't think so, merging now...

**Pull request merged**